events.html

```diff
@@ -53,9 +53,10 @@
             <h1>Upcoming Events</h1>
             <div class="event-list">
                 <a class="event" href="https://www.eventbrite.ca/e/frostycon-csc-game-night-7-tickets-31724885988">
-                    <h3>IBM Hackathon & CSC Lan Party #7<b>iTel</b></h3>
-                    <h5>February 11th - 9:00am to February 12th - 2:00am</h5>
-                    Day Hackathon with IBM Watson/Bluemix from 9:00am to 5:00pm (lunch not provided). Boardgames and LAN setup 5:00 pm to 7:30pm, LAN Party 7:30pm to 2:00am, and Pizza at 8:30pm.
+                    <h3>IBM Hackathon &amp; CSC Lan Party #7 @ <b>iTel</b></h3>
+                    <h5>9:00am Saturday, February 11th to 2:00am Sunday, February 12th</h5>
+                    Starting 9:00am on Saturday we will be hosting a all day Hackathon with IBM's Watson. First we will be creating interfaces with Watson using their Bluemix API with Node.js and Android studio.
```

Comment:
"...hosting a**n** all day **h**ackathon...".
"Then **,** teams of 1 to 4 will hack together a project before 4:00pm."